package.json

```diff
@@ -54,32 +54,48 @@
   "additionalDependencies": {
     "babel-core": "6.24.0",
     "babel-loader": "6.2.4",
-    "babel-preset-es2015": "6.9.0",
-    "babel-preset-react": "6.11.1",
-    "babel-preset-stage-2": "6.11.0",
     "copy-webpack-plugin": "3.0.1",
     "css-loader": "0.23.1",
-    "css-to-string-loader": "0.1.0",
-    "extract-text-webpack-plugin": "1.0.1",
     "file-loader": "0.9.0",
     "html-loader": "0.4.3",
-    "html-webpack-plugin": "2.22.0",
+    "html-webpack-plugin": "2.28.0",
     "null-loader": "0.1.1",
     "postcss-loader": "0.9.1",
     "postcss-cssnext": "2.9.0",
-    "postcss-import": "9.1.0",
+    "postcss-url": "5.1.2",
     "progress-bar-webpack-plugin": "1.8.0",
     "raw-loader": "0.5.1",
-    "react-hot-loader": "3.0.0-beta.1",
     "read-cache": "1.0.0",
     "stylus": "0.54.5",
     "stylus-loader": "2.1.1",
-    "style-loader": "0.13.1",
-    "ts-loader": "1.3.0",
-    "typescript": "2.0.10",
-    "vue-loader": "11.0.0",
-    "vue-template-compiler": "2.2.6",
+    "style-loader": "0.13.1"
+  },
+  "dependenciesAngular2": {
```

Comment:
The naming convention needs to be consistent: FrameworkName + "Dependencies`,  like `wepack2Dependencies`, `additionalDependencies` etc. Please fix all the inconsistent ones.